content/docs/command-reference/exp/diff.md

```diff
@@ -0,0 +1,39 @@
+# exp diff
+
+Show [parameters](/doc/command-reference/params) and
+[metrics](/doc/command-reference/metrics) changes between `dvc experiments`, or
+between the last <abbr>project</abbr> version and the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc experiments diff [-h] [-q | -v] [--all] [--show-json]
+                            [--show-md] [--old] [--no-path]
+                            [--precision <n>] [a_rev] [b_rev]
+
+positional arguments:
+  a_rev            Old experiment to compare (defaults to HEAD)
+  b_rev            New experiment to compare (defaults to the current
+                   workspace)
+```
+
+# Description
+
+Provides a quick way to compare `dvc params` and `dvc metrics` between two
+`dvc experiments`. The differences shown by this command include the old and new
+values, and numeric difference (delta) from the previous ones.
+
+> This is similar to combining the reports from `dvc params diff` and
+> `dvc metrics diff` together, for the experiments in question.
+
+Without arguments, this command compares params and metrics currently present in
+the <abbr>workspace</abbr> (last `dvc exp run`) with the latest committed
+versions (required).
+
+`a_rev` and `b_rev` are optional experiment IDs (printed by `dvc exp run`) to
+compare.
+
+All params and metrics defined in `dvc.yaml` are used by default.
```

Comment:
Are params/metrics that only come up in one experiment shown?

Reply:
Not sure I understand this question, it will show all defined params/metrics from both `a_rev` and `b_rev`, the same way that it is handled in metrics/params diff

Reply:
also, same as above, the target to compare doesn't have to be an experiment name, it can be any git revision or tag/branch

Reply:
Sorry, I wasn't clear. I meant if a param/metric is defined in one of the revs but not in the other what happens? I guess it behaves like the each `diffs` where `-` is displayed in the missing one (and no change of course). Prob. no need to document it anyway, resolving!